tests/Utils/CoerceValueTest.php

```diff
@@ -76,17 +87,23 @@ public function testCoercingAnArrayToGraphQLIDProducesAnError(): void
      * Describe: for GraphQLInt.
      *
      * @param mixed $result returned result
+     * @param InputPath|null $expectedPath
      */
-    private function expectGraphQLError($result, string $expected): void
+    private function expectGraphQLError($result, string $expected, ?array $expectedPath = null): void
     {
         self::assertIsArray($result);
 
         $errors = $result['errors'];
         self::assertIsArray($errors);
         self::assertCount(1, $errors);
+        self::assertInstanceOf(CoercionError::class, $errors[0]);
         self::assertSame($expected, $errors[0]->getMessage());
 
         self::assertNull($result['value']);
+
+        if ($expectedPath !== null) {
+            self::assertSame($expectedPath, $errors[0]->inputPath);
+        }
```

Comment:
Shouldn't those be equivalent even when `null`?

Reply:
Removed check.